.gitignore

```diff
@@ -1,6 +1,6 @@
 # project artifcats
-lib
 docs
+lib
```

Comment:
Is there a specific reason for switching those two lines?

Reply:
Nope, this was accidental. I temporarily add `lib` as a tracked directory on one of my fork's branches to enable installing from git while I was working on these changes locally.

Will revert to clean things up :)